src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -175,7 +175,9 @@ private IEnumerable<NuGet.Credentials.ICredentialProvider> GetCredentialProvider
         {
             var extensionLocator = new ExtensionLocator();
             var providers = new List<Credentials.ICredentialProvider>();
-            var pluginProviders = new PluginCredentialProviderBuilder(extensionLocator, Settings).BuildAll().ToList();
+            var pluginProviders = new PluginCredentialProviderBuilder(extensionLocator, Settings, Console)
+                .BuildAll(Verbosity.ToString())
```

Comment:
I addressed the first suggestion -- we won't send "-verbosity normal" to plugins, since this should be the default.

re: the enum, after discussing with @emgarten, will leave as a string for now.